portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,66 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+h1 {
+    color: black;
+    font-size: 55px;
+    font-weight: bold;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
```

Comment:
I think we should keep indentation inside these blocks consistent! For example right now these 2 blocks are indented differently. Pick one and then align indentation for the remaining blocks in the file.